### DISCUSSION TITLE
Install Problem: No Python at '"D:\Program Files\Python\python.exe'

### DISCUSSION THREAD

**Discussion by riveter1** [bmaltais/kohya_ss#1306] [Q&A]
I had Python installed on D: drive but uninstalled it and reinstalled Python to the standard location on C: drive. I did let Python update the Paths env variable. I rebooted to PC.
I cloned kohya_ss to the D: drive and ran setup.bat as admin.
I get this message: python: can't open file 'C:\\Windows\\System32\\setup\\check_local_modules.py': [Errno 2] No such file or directory
No Python at '"D:\Program Files\Python\python.exe'
Any ideas to fix this?

**Comment by MahsaBakhtiari** [comment]
It looks like your kohya_ss setup is having trouble finding Python on your C: drive after you uninstalled and reinstalled it. Here's what you can do:
1. Check if Python is installed correctly on C: drive.
2. Verify that the environment variables are updated with the new Python path.
3. Remove any old paths to the previous Python installation.
4. Update kohya_ss settings to use the new Python path.
5. Run setup.bat as admin to give it the necessary permissions.
6. If the issue persists, check kohya_ss documentation or seek help from its support.
That should help you get things running smoothly again.

**Reply by TeeFReUnD** [reply]
Hey, i got the same problem and currently stuck at step 4. How do i change or update kohya_ss settings to use the new Python path? Which file do i need to update? In gui.bat it only says "set PATH=%PATH%". I already tried to change %PATH% to "C:\Users\x\AppData\Local\Programs\Python\Python310\", which is listed in system path settings, but this does not work. When i try to run setup.bat, i get the message "No python at "c:\Python311\python.exe".

I´m quite confused, how to fix this.

Appreciate any help :)

**Reply by jetzwow** [reply]
The settings file is located at .\kohya_ss\venv\pyvenv.cfg .

Change the old path to the new one.

**Comment by Shadeling3D** [comment]
I've been at this for days. this is all I get trying to set up Kohya_ss.

![Python](https://github.com/bmaltais/kohya_ss/assets/32183085/bf4e7b5a-d35c-4523-acad-87e597db44b7)

![setup](https://github.com/bmaltais/kohya_ss/assets/32183085/07e281cc-886c-4804-8121-9ae5677e177f)

**Reply by Shadeling3D** [reply]
I have deleted and re-acquired multiple times. Both SDXL and Comfy are working perfectly, Kohya SS is my problem child. 
Windows 11 on a Razer Blade 15 Base with maxed out memory and an additional 1TB SSD installed.
